src/org/spdx/htmltemplates/LicenseHTMLFile.java

```diff
@@ -73,24 +78,58 @@ public void setUrl(String url) {
 		public String getSite() {
 			return getSiteFromUrl(url);
 		}
+		
+		public String getVal(Integer index) {
+			String val = null;
+			String currentUrl = null;
+			if(license != null) {
+				for(String ref: license.getCrossRef()) {
+					String crossRef = ref.substring(1, ref.length()-1);
+					String[] details = crossRef.split(",");
+					currentUrl = details[0].split(": ")[1].trim();
+					if(url.equals(currentUrl)) {
+						val = details[index].split(": ")[1].trim();
+					}
+				}
+			}
+			return val;
+		}
 
 		public boolean getIsValid() {
+			if(license != null) {
+				boolean b = Boolean.parseBoolean(getVal(1));
```

Comment:
@tjasmith can you change these to a constants to improve readability - e.g. `getVal(CROSS_REF_INDEX_ISVALID)`